apps/documentation/src/routes/documentation/installation.mdx

```diff
@@ -13,25 +13,25 @@
 
 ## Requirements
 
-`Tuono` is a development environment built in rust and typescript that outputs a website written with both the
-languages. Since that you need the following tools installed on your computer to work with it:
+`Tuono` is a development environment built in rust and typescript that outputs a website written in both the
+languages. Since then, you need the following tools installed on your computer to work with it:
 
-- `Rust` - The rust programming language toolchain (Go [here](https://rustup.rs/) for installing both `rust` and `cargo`)
+- `Rust` - The Rust programming language tool chain (Go [here](https://rustup.rs/) for installing both `rust` and `cargo`)
 - `Cargo` - The rust package manager
-- `NodeJs` - The Javascript runtime (Go [here](https://nodejs.org/en/download/package-manager) for installing both `nodejs` and `npm`)
-- `npm` or `yarn` or `pnpm` - A javascript package manager
+- `NodeJs` - The JavaScript runtime (Go [here](https://nodejs.org/en/download/package-manager) for installing both `nodejs` and `npm`)
+- `npm` or `yarn` or `pnpm` - A javaScript package manager
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Possible spelling mistake found. (MORFOLOGIK_RULE_EN_US)
Suggestions: `NPM`, `PM`, `pm`, `rpm`, `NP`, `NPC`, `NPR`, `RPM`, `ppm`, `NPA`, `NPS`, `Nam`, `APM`, `BPM`, `CPM`, `NPD`, `TPM`, `LPM`, `GPM`, `wpm`, `EPM`, `FPM`, `IPM`, `MPM`, `NAM`, `NCM`, `NEM`, `NKM`, `NM`, `NMM`, `NP0`, `NPF`, `NPG`, `NPI`, `NPL`, `NPN`, `NPO`, `NPP`, `NPV`, `NRM`, `NSM`, `NVM`, `Np`, `OPM`, `PPM`, `Pm`, `QPM`, `SPM`, `XPM`, `bpm`, `nm`, `n pm`, `not`, `him`, `new`, `name`, `no`, `now`, `up`, `km`, `non`, `FM`, `NFL`, `am`, `arm`, `cm`, `com`, `mm`, `nor`, `IBM`, `Jim`, `PA`, `PC`, `Tom`, `mph`, `net`, `sum`, `CPU`, `NSW`, `Sam`, `gym`, `neo`, `norm`, `rim`, `spy`, `ups`, `GPS`, `HP`, `MP3`, `NFC`, `NGO`, `NHS`, `NRL`, `NYC`, `PR`, `PT`, `RAM`, `RPG`, `dam`, `gum`, `nun`, `nut`, `opt`, `pH`, `ram`, `spa`, `ACM`, `APA`, `APC`, `ATM`, `BNP`, `CPA`, `CPC`, `CPI`, `CPR`, `DM`, `DRM`, `DSM`, `EPs`, `GPA`, `GPL`, `Ham`, `IPA`, `IPO`, `JP`, `LPs`, `NAD`, `NBL`, `NEC`, `NGC`, `NMR`, `NSA`, `NYPD`, `Nov`, `OEM`, `PD`, `PPP`, `PV`, `SAM`, `SNP`, `SPD`, `VP`, `WPA`, `amp`, `ape`, `apt`, `cam`, `dim`, `elm`, `gem`, `ham`, `mom`, `nod`, `nu`, `pa`, `pi`, `rum`, `spam`, `APG`, `APS`, `BPA`, `CPP`, `DTM`, `EPP`, `FP`, `GNP`, `GPO`, `GPU`, `HPV`, `IPC`, `IPS`, `KLM`, `KTM`, `LPG`, `MPA`, `MPH`, `MPP`, `NAC`, `NCA`, `NCO`, `NCR`, `NDA`, `NEA`, `NLP`, `NPCs`, `NRA`, `NRC`, `NSF`, `Napa`, `Nat`, `PCM`, `PJ`, `PNP`, `PPD`, `PPI`, `Pa`, `Pam`, `REM`, `RPC`, `RPF`, `SPL`, `UPC`, `UPI`, `UPS`, `VM`, `VPN`, `hum`, `mum`, `nape`, `nee`, `nil`, `ops`, `um`, `yam`, `ABM`, `AFM`, `APD`, `Apr`, `BPD`, `BRM`, `CBM`, `CPE`, `CPO`, `CTM`, `DCM`, `DPI`, `DWM`, `EAM`, `EPG`, `ERM`, `FSM`, `GPC`, `ICM`, `IOM`, `IPN`, `IPP`, `IPR`, `JDM`, `JM`, `JVM`, `LSM`, `MPR`, `MPV`, `NBS`, `NCI`, `NDC`, `NEP`, `NFPA`, `NFS`, `NIR`, `NIV`, `NLS`, `NRG`, `NRI`, `NTP`, `NWT`, `NY`, `Nome`, `PMA`, `PMC`, `PMI`, `PMR`, `PMS`, `PSM`, `PW`, `RCM`, `RPA`, `RPI`, `RPO`, `RPR`, `RSM`, `SNPs`, `SPF`, `SRM`, `SSM`, `TCM`, `TPA`, `TPC`, `Ulm`, `bum`, `hem`, `imp`, `mpg`, `nap`, `nay`, `numb`, `ohm`, `ppb`, `pt`, `rem`, `tam`, `APB`, `APN`, `Adm`, `BIM`, `CPF`, `DLM`, `DPD`, `DPT`, `EPL`, `FAM`, `FPA`, `GAM`, `GBM`, `IPI`, `KVM`, `LCM`, `LPO`, `LPP`, `MDM`, `MNP`, `NAL`, `NDB`, `NFF`, `Noe`, `PNM`, `PX`, `Qom`, `RPN`, `RPT`, `SDM`, `SGM`, `Spam`, `aim`, `fem`, `fps`, `gm`, `mp`, `nag`, `nip`, `nth`, `tum`, `yum`, `DPN`, `DPs`, `DVM`, `EPF`, `FPO`, `GPV`, `LPN`, `MPE`, `NLR`, `NVR`, `Nev`, `RMM`, `TPL`, `TPN`, `UAM`, `WLM`, `bam`, `bps`, `dBm`, `nab`, `nae`, `naps`, `nit`, `nix`, `nub`, `pk`, `PMs`, `app`, `fum`, `kph`, `mam`, `neg`, `nips`, `nus`, `opp`, `vim`, `Jpn`, `nob`, `nope`, `GM`, `IP`, `Kim`, `MPs`, `NBA`, `NBC`, `NDP`, `NHL`, `Ned`, `PS`, `ROM`, `Tim`, `chm`, `cpl`, `dpt`, `jam`, `née`, `op`, `rps`, `cpd`, `ppr`, `2RM`, `A&M`, `A6M`, `AAM`, `ADM`, `AEM`, `AGM`, `AGPM`, `AHM`, `AIM`, `AKM`, `ALM`, `AM`, `AMM`, `ANM`, `ANP`, `ANPE`, `ANPN`, `ANPS`, `ANSM`, `AOM`, `AP`, `APE`, `APF`, `APH`, `API`, `APJ`, `APK`, `APL`, `APO`, `APP`, `APQ`, `APR`, `APT`, `APU`, `APV`, `APW`, `APX`, `APY`, `APZ`, `APs`, `AQM`, `ARM`, `ASM`, `AUM`, `AVM`, `AWM`, `AXM`, `Am`, `B2M`, `BAM`, `BBM`, `BCM`, `BDM`, `BEM`, `BFM`, `BGM`, `BHM`, `BIPM`, `BJM`, `BKM`, `BLM`, `BM`, `BMM`, `BNM`, `BOM`, `BP`, `BPB`, `BPC`, `BPF`, `BPG`, `BPH`, `BPI`, `BPK`, `BPL`, `BPMG`, `BPMs`, `BPN`, `BPO`, `BPP`, `BPR`, `BPS`, `BPT`, `BPU`, `BPY`, `BSM`, `BTM`, `BUM`, `BVM`, `BWM`, `BXM`, `BYM`, `CAM`, `CCM`, `CDM`, `CEM`, `CFM`, `CGM`, `CHM`, `CIM`, `CKM`, `CLM`, `CM`, `CMM`, `CNAM`, `CNM`, `CNP`, `CNPC`, `CNPE`, `CNPN`, `CNPS`, `COM`, `CP`, `CPAM`, `CPD`, `CPG`, `CPH`, `CPJ`, `CPL`, `CPMs`, `CPN`, `CPQ`, `CPS`, `CPT`, `CPV`, `CQM`, `CRM`, `CSM`, `CUM`, `CVM`, `CWM`, `CXM`, `CYM`, `CZM`, `Cm`, `Com`, `Cpl`, `DDM`, `DEM`, `DFM`, `DHM`, `DMM`, `DNP`, `DOM`, `DP`, `DPA`, `DPB`, `DPC`, `DPE`, `DPH`, `DPMI`, `DPO`, `DPS`, `DPU`, `DPW`, `DXM`, `Dem`, `ECM`, `EDM`, `EEM`, `EM`, `EMM`, `ENIM`, `ENM`, `ENP`, `ENPC`, `ENSM`, `ENSPM`, `EOM`, `EP`, `EPA`, `EPCM`, `EPI`, `EPO`, `EPR`, `EPS`, `EPSM`, `EPT`, `EPU`, `EPV`, `EQM`, `ESM`, `ETM`, `EVM`, `FBM`, `FCM`, `FDM`, `FEM`, `FFM`, `FFPM`, `FGM`, `FHM`, `FIM`, `FKM`, `FLM`, `FMM`, `FNPC`, `FNPF`, `FPD`, `FPF`, `FPH`, `FPJ`, `FPP`, `FPR`, `FPS`, `FPT`, `FPU`, `FPV`, `FPs`, `FRM`, `FTM`, `Fm`, `GCM`, `GDM`, `GEM`, `GHM`, `GLM`, `GP`, `GPE`, `GPG`, `GPP`, `GPR`, `GPT`, `GPX`, `GPZ`, `GQM`, `GSM`, `GTM`, `GUM`, `H&M`, `HBM`, `HDM`, `HLM`, `HM`, `HOM`, `HPA`, `HPC`, `HPD`, `HPE`, `HPL`, `HPR`, `HPS`, `HRM`, `HSM`, `IDM`, `IEM`, `IFM`, `IGM`, `IIM`, `IJM`, `ILM`, `IM`, `INM`, `INP`, `INPP`, `INPT`, `IPAM`, `IPB`, `IPD`, `IPE`, `IPF`, `IPG`, `IPJ`, `IPT`, `IPY`, `IPs`, `IRM`, `ISM`, `JIM`, `JLM`, `JMM`, `JPA`, `JPF`, `JPG`, `JPN`, `JPO`, `JPP`, `JPR`, `JPX`, `JPY`, `Jem`, `KEM`, `KFM`, `KGM`, `KHM`, `KM`, `KNM`, `KP`, `KPD`, `KPF`, `KPI`, `KPL`, `KPMG`, `KPN`, `KPU`, `KPW`, `KSM`, `LBM`, `LEM`, `LFM`, `LGM`, `LIM`, `LKM`, `LNPA`, `LP`, `LPC`, `LPD`, `LPE`, `LPF`, `LPI`, `LPJ`, `LPQ`, `LPR`, `LPS`, `LRM`, `Lim`, `M`, `M&M`, `M2M`, `MAM`, `MCM`, `MFM`, `MGM`, `MIM`, `MKM`, `MLM`, `MM`, `MMM`, `MOM`, `MP`, `MP4`, `MPC`, `MPD`, `MPI`, `MPL`, `MPN`, `MPS`, `MPT`, `MPX`, `MSM`, `MZM`, `N`, `N10`, `N11`, `N12`, `N13`, `N14`, `N15`, `N16`, `N17`, `N18`, `N19`, `N20`, `N21`, `N22`, `N23`, `N24`, `N25`, `N26`, `N27`, `N28`, `N29`, `N30`, `N31`, `N32`, `N33`, `N34`, `N35`, `N40`, `N41`, `N42`, `N43`, `N44`, `N45`, `N48`, `N49`, `N50`, `N51`, `N52`, `N53`, `N61`, `N62`, `N63`, `N64`, `N70`, `N71`, `N72`, `N73`, `N74`, `N75`, `N76`, `N77`, `N78`, `N79`, `N80`, `N81`, `N82`, `N85`, `N86`, `N90`, `N91`, `N92`, `N93`, `N95`, `N96`, `N97`, `N99`, `NAB`, `NAF`, `NAG`, `NAH`, `NAK`, `NAMC`, `NAO`, `NAP`, `NAS`, `NAU`, `NB`, `NBB`, `NBF`, `NBG`, `NBN`, `NBR`, `NBT`, `NC`, `NCB`, `NCC`, `NCD`, `NCE`, `NCL`, `NCP`, `NCQ`, `NCS`, `NCT`, `NCW`, `NCY`, `ND`, `NDF`, `NDH`, `NDK`, `NDMP`, `NDR`, `NDS`, `NDT`, `NE`, `NED`, `NEE`, `NEH`, `NEN`, `NER`, `NF`, `NF3`, `NFT`, `NG`, `NGA`, `NGE`, `NGN`, `NGS`, `NH`, `NHA`, `NHC`, `NHD`, `NHF`, `NHK`, `NHN`, `NI`, `NIA`, `NIB`, `NIC`, `NID`, `NIF`, `NIG`, `NIH`, `NIN`, `NIO`, `NIU`, `NJ`, `NJG`, `NJP`, `NK`, `NL`, `NLB`, `NLD`, `NLG`, `NLH`, `NLO`, `NLT`, `NLU`, `NMA`, `NMC`, `NMF`, `NMI`, `NMPP`, `NMT`, `NNI`, `NNN`, `NO`, `NOA`, `NOE`, `NOI`, `NOK`, `NON`, `NOPD`, `NOR`, `NOS`, `NOU`, `NOW`, `NOX`, `NPAG`, `NPAI`, `NPHC`, `NPNS`, `NPOV`, `NQY`, `NR`, `NRF`, `NRH`, `NRN`, `NRO`, `NRP`, `NRS`, `NRT`, `NRU`, `NRW`, `NS`, `NSB`, `NSBM`, `NSC`, `NSS`, `NST`, `NSU`, `NT`, `NTC`, `NTD`, `NTI`, `NTL`, `NTPs`, `NTR`, `NTS`, `NTV`, `NTW`, `NU`, `NUI`, `NUMA`, `NUS`, `NUT`, `NV`, `NVA`, `NVMe`, `NVP`, `NVT`, `NW`, `NWD`, `NYH`, `NYK`, `NYS`, `NYT`, `NYU`, `NYX`, `NZ`, `NZD`, `NZL`, `Na`, `Nan`, `Nb`, `Nd`, `Ne`, `Neb`, `Nemo`, `Neo`, `Ni`, `No`, `Nos`, `Nye`, `O&M`, `OAM`, `ODM`, `OGM`, `OHM`, `OIM`, `OM`, `OMM`, `ONEM`, `ONM`, `ONP`, `OOM`, `OPA`, `OPG`, `OPI`, `OPK`, `OPL`, `OPML`, `OPO`, `OPQ`, `OPT`, `ORM`, `OSM`, `OVPM`, `P`, `PAM`, `PB`, `PE`, `PEM`, `PG`, `PGM`, `PH`, `PIM`, `PK`, `PL`, `PLM`, `PMB`, `PMD`, `PME`, `PMF`, `PMG`, `PMH`, `PML`, `PMN`, `PMO`, `PMP`, `PMT`, `PMU`, `PMV`, `PO`, `POM`, `PP`, `PP1`, `PP2`, `PP3`, `PP4`, `PP7`, `PPA`, `PPB`, `PPC`, `PPCM`, `PPE`, `PPH`, `PPK`, `PPL`, `PPN`, `PPO`, `PPQ`, `PPR`, `PPS`, `PPT`, `PPU`, `PPV`, `PQ`, `PRM`, `PTM`, `PU`, `PUM`, `PVM`, `PWM`, `Pb`, `Pd`, `Pl`, `Po`, `Pr`, `Pt`, `Pu`, `Pym`, `QCM`, `QM`, `QPC`, `QPO`, `QPV`, `RBM`, `RDM`, `RFM`, `RIM`, `RM`, `RP`, `RPB`, `RPD`, `RPE`, `RPK`, `RPP`, `RPS`, `RPU`, `RPV`, `RPX`, `RTM`, `RVM`, `Rom`, `SBM`, `SCM`, `SEM`, `SFM`, `SHM`, `SIM`, `SJM`, `SM`, `SMM`, `SNCM`, `SNIM`, `SNSM`, `SP`, `SP1`, `SPA`, `SPB`, `SPC`, `SPE`, `SPG`, `SPH`, `SPI`, `SPP`, `SPR`, `SPS`, `SPT`, `SPV`, `SPW`, `STM`, `SVM`, `SXM`, `Sm`, `Sp`, `TAM`, `TBM`, `TDM`, `TEM`, `THM`, `TKM`, `TLM`, `TM`, `TNM`, `TNP`, `TOM`, `TP`, `TPE`, `TPF`, `TPG`, `TPI`, `TPJ`, `TPO`, `TPP`, `TPR`, `TPS`, `TPT`, `TPV`, `TPX`, `TQM`, `TRM`, `TSM`, `TVM`, `Tm`, `UCM`, `UDM`, `UEM`, `UGM`, `UIM`, `UJM`, `ULM`, `UMM`, `UNAM`, `UNM`, `UNPO`, `UP`, `UP1`, `UPA`, `UPB`, `UPD`, `UPE`, `UPF`, `UPK`, `UPMC`, `UPN`, `UPP`, `UPR`, `UPU`, `UPV`, `UTM`, `UVM`, `VAM`, `VCM`, `VDM`, `VGM`, `VLM`, `VMM`, `VNM`, `VPC`, `VPE`, `VPH`, `VPI`, `VPP`, `VPS`, `VPs`, `VSM`, `WAM`, `WDM`, `WFM`, `WHM`, `WIM`, `WP`, `WPC`, `WPF`, `WPP`, `WPS`, `WSM`, `Wm`, `XP`, `XPF`, `XPS`, `YEM`, `YM`, `YP`, `YQM`, `YUM`, `ZAM`, `ZIM`, `ZNP`, `ZPE`, `ZPP`, `cps`, `cum`, `dm`, `dpi`, `em`, `hm`, `hmm`, `hp`, `ism`, `lam`, `m`, `n`, `nah`, `nav`, `nib`, `nos`, `né`, `om`, `p`, `pc`, `pd`, `pf`, `pg`, `pj`, `pl`, `pms`, `pom`, `pp`, `ppl`, `pr`, `rm`, `sim`, `tmp`, `tnpk`, `tom`, `uhm`, `umm`, `ump`, `vm`, `µm`, `3M`, `LVM`, `NTT`, `NXP`, `TPU`
Rule: https://community.languagetool.org/rule/show/MORFOLOGIK_RULE_EN_US?lang=en-US
Category: TYPOS
<!-- __reviewdog__:ChBkYzhkMTIzNjIzYmVmMTQ5EgxMYW5ndWFnZVRvb2w= -->